AndelaOSP/Andela-Socials
# #165850583 Regularise fonts across application

#### What Does This PR Do?
  This PR regularizes the font in the application to match Andela conventions

#### Description Of Task To Be Completed
    - Add Lato font to description body
    - Format some part of code to follow eslint rules

#### Any Background Context You Want To Provide?
   None

#### How can this be manually tested?
  Navigate to the event details page, there you will see the description body using Lato

#### What are the relevant pivotal tracker stories?
[#165850583](https://www.pivotaltracker.com/story/show/165850583)

#### Screenshot(s)
N/A
